perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2795,58 +2842,88 @@ perun_policies:
         SELF: User
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   setSponsoredMember_Vo_User_String_String_User_LocalDate_policy:
     policy_roles:
       - SPONSOR: Vo
         SELF: User
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Member
+      - MFA: Vo
 
   createSponsoredMembersFromCSV_Vo_String_List<String>_User_policy:
     policy_roles:
       - SPONSOR: Vo
         SELF: User
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Member
```

Comment:
```suggestion
```

Reply:
Not sure what Member object would be checked - there is none before the method is called